src/Components/Admin/Rooms/roomModal.js

```diff
@@ -60,6 +73,31 @@ export default function RoomModal({
         );
     }, [room]);
 
+    const handleUploadFileChange = (event) => {
+        setErrors({ ...errors, assetUpload: "" });
+        if (event.target.files && event.target.files[0]) {
+            const file = event.target.files[0];
+            const name = file.name;
+            const lastDot = name.lastIndexOf(".");
+            const split = name.split(lastDot);
+            const fileName = split[0];
+            const fileType = split[1];
+
+            setFileName(fileName);
+            handleUploadAssetSubmit(fileName, fileType, file);
+        }
+    };
+
+    const handleUploadAssetSubmit = async (name, file_type, asset) => {
+        const response = await createPresignedLinkAndUploadS3(
+            { file_type: file_type, type: "asset", file_content: asset },
```

Comment:
Again this is not correct: the type shouldn't be an `asset`. It will store the image in the wrong prefix in S3. This should be type `image`. See Amolik's implementation for this: https://github.com/uwblueprint/cdc-frontend/blob/a8f1e94b58e04c5cfb0ffafe15690da554431b44/src/Components/Admin/ObjectEditor/visualpaneview.js#L44